api/src/test/resources/io/kaoto/backend/api/resource/eip.kamelet.yaml

```diff
@@ -45,8 +45,7 @@ spec:
                   sample-period: '1500'
                   message-frequency: 5
               - script:
-                  expression:
```

Comment:
Script should support expression, why remove it?

 https://github.com/apache/camel/blob/main/dsl/camel-yaml-dsl/camel-yaml-dsl/src/test/groovy/org/apache/camel/dsl/yaml/ScriptTest.groovy

Reply:
We can add like `ScriptExpression` and do the same thing with `Expression` but also keep script expression isolated from other expression. but...

It reminded me that in the past, you said we need to support it from step extensions as well, and I haven't worked on it yet. Is it considered as a bug? Is there a issue for that? It'd need to be created or added to Sprint 16 I guess.